src/helpers/appBar.js

```diff
@@ -1,5 +1,10 @@
-import React from "react";
+import React, { useEffect, useState, useContext } from "react";
```

Comment:
Hi, we can remove this imports right?

Reply:
yes, Updated the file.

Reply:
@rijusougata13 I did the suggested changes, can you please review?

Reply:
hi, can you post a screenshot for both the cases 1. when user signed in 2. not signed in

Reply:
Case 1. When User is signed In 
![image](https://user-images.githubusercontent.com/55620053/152188373-861f2ebe-8490-4df0-aa0b-9d558aedb998.png)

Case 2. When User is not signed in
![image](https://user-images.githubusercontent.com/55620053/152188843-bce224f9-2a06-4fd7-af32-6aa5dca9e2ba.png)
![image](https://user-images.githubusercontent.com/55620053/152188875-ec2b8224-8164-4eb8-a5f4-4fb09e6063d5.png)
![image](https://user-images.githubusercontent.com/55620053/152188926-0282eada-e7ed-4530-b7cf-33bebd96da6c.png)